### DISCUSSION TITLE
4chpror3 how reset ap mode

### DISCUSSION THREAD

**Discussion by songzh96** [arendst/Tasmota#18106] [Q&A]
Hi
I entered the wrong wifi information, which makes it impossible to control now. How do I reset the ap mode】

Best Regards

**Comment by Jason2866** [comment]
Tried 40 seconds button press?

**Reply by songzh96** [reply]
not working....

**Comment by HofeBY** [comment]
six times Power remove and 10sec later power on maybe bring a Device in Access Mode
look to WLAN Networks. tasmota.....

**Reply by songzh96** [reply]
When I powered on, I pressed the button 6 times within 10s, and pressed it for 10s for the 7th time, but it didn't work.

**Reply by barbudor** [reply]
Not what was described
You need to power off and on the Sonoff device, not the relay
Cut mains from the device, wait 2 sec, apply mains, wait 2 sec, and again 6 more times
That should reset the configuration

Or press Button 1 for more than 40 seconds

See doc
https://tasmota.github.io/docs/Device-Recovery/
